src/simplicial-hott/03-extension-types.rzk.md

```diff
@@ -1093,41 +1093,41 @@ f(t) = a'(t) \biggr|^\phi_{\lambda t.refl} \right\rangle$
         ( t)))
     ( RS-4-11-c
       htpy-ext-prop I ψ ϕ A a f is-contr-fiberwise-A)
-    ( \ t → refl )
+    ( \ t → refl)
     ( RS-4-11-c-is-refl I ψ ϕ A
       ( is-contr-fiberwise-A)
-      ( a )
-      ( f )
+      ( a)
+      ( f)
       ( first
         ( htpy-ext-prop-is-fiberwise-contr
           htpy-ext-prop
           I ψ ϕ A a
           is-contr-fiberwise-A))
       ( RS-4-11-c
         ( htpy-ext-prop)
-        ( I) ( ψ) ( ϕ) ( A) ( a) ( f)
-        ( is-contr-fiberwise-A ))))
+        ( I) (ψ) (ϕ) (A) (a) (f)
+        ( is-contr-fiberwise-A))))
 ```
 
 ```rzk title="RS17, Proposition 4.11"
 #define weak-extext-naiveextext-htpy-ext-property
  ( naiveextext : NaiveExtExt)
  ( htpy-ext-prop : HtpyExtProperty)
- : WeakExtExt
+  : WeakExtExt
   := \ I ψ ϕ A is-contr-fiberwise-A a →
-    (first (htpy-ext-prop-is-fiberwise-contr htpy-ext-prop I ψ ϕ A a is-contr-fiberwise-A),
-     \ f →
+    ( first (htpy-ext-prop-is-fiberwise-contr htpy-ext-prop I ψ ϕ A a is-contr-fiberwise-A)
```

Comment:
Since the definition is more than one line long, a newline should be inserted directly after `:=` (also note that the indentation is wrong in this example)